draft-ietf-core-echo-request-tag.md

```diff
@@ -189,7 +197,7 @@ The CoAP server side of CoAP-to-HTTP proxies MAY request freshness, especially i
 
     * For actuator applications with low delay tolerance, to avoid additional round-trips for multiple requests in rapid sequence, the server may include the Echo option with a new value even in a successful response to a request, irrespectively of whether the request contained an Echo option or not. The client then uses the Echo option with the new value in the next actuation request, and the server compares the receive time accordingly.
 
```

Comment:
"E.g. if access to a server resource is dependent on time, then the client requesting access MUST NOT set the time of the server."

Two comments of this sentence.
1. This seems to rely on that the server is not the authority (which is may be)
1. A client syncronizing seems to have nothing to do with echo